documentation/src/main/asciidoc/user_guide/chapter-76-Simple_Cache.adoc

```diff
@@ -0,0 +1,47 @@
+== Simple Cache
+
+Traditional local caches use the same architecture as clustered caches - interceptor stack. That way a lot of the functionality can be reused. However, if the advanced features are not needed and the performance is more important, the interceptor stack can be stripped away and simple cache can be used.
+
+So, which features are stripped away? From the configuration perspective, simple cache does not support:
+
+* transactions and invocation batching
+* persistence (cache stores and loaders)
+* custom interceptors (there's no interceptor stack!)
+* indexing
+* compatibility (embedded/server mode)
+* store as binary (which is hardly useful for local caches)
+
+From the API perspective these features throw an exception:
+
+* adding custom interceptors
+* Map Reduce Framework
+* Distributed Executors Framework
```

Comment:
I wouldn't call the distributed executor a framework. Maybe the name fits a little better for Map/Reduce, because the user has to write multiple functions that work together, but I'd say it's a stretch there as well.